src/components/Dialog.scss

```diff
@@ -1,6 +1,10 @@
 @import "../css/_variables";
 
 .excalidraw {
+  .Dialog {
+    user-select: text;
```

Comment:
```suggestion
    user-select: text;
    cursor: auto;
```

Reply:
good catch on the `text` cursor. Should be set to `auto` instead.

Reply:
Thank you @dwelle. I have done the change now. should I squash the commits before merge?

Reply:
no.. we squash before merging